docs/source/use_cases.rst

```diff
@@ -6,13 +6,13 @@ Use case #1 - Spinal Cord Toolbox:
 
 `Spinal cord toolbox <http://spinalcordtoolbox.com/>`__ (SCT) is an open-source analysis software package for processing MRI data of the spinal cord `[De Leener et al. 2017] <https://doi.org/10.1016/j.neuroimage.2016.10.009>`__. `ivadomed` is SCT's backbone for the automated segmentation of the spinal cord, gray matter, tumors, and multiple sclerosis lesions, as well as for the labeling of intervertebral discs.
 
-Use case 2 - Creation of anatomical template:
----------------------------------------------
+Use case #2 - Creation of anatomical template:
```

Comment:
no need for "use case 1", etc.
also please remove the ":"

Reply:
Fixed in [99aa381](https://github.com/ivadomed/ivadomed/pull/1080/commits/99aa38104f7e2777315b0ea04e3cb871938838a9)